packages/product-json-to-csv/src/map-product-data.js

```diff
@@ -263,13 +263,17 @@ export default class ProductMapping {
 
   static _mapPriceToString(price: Price): string {
     // Full price:
-    // 'country-currencyCode centAmount|discounted.centAmount customerGroup.name#channel.key$validFrom~validUntil'
+    // 'country-currencyCode centAmount|discounted.centAmount@customerGroup.name#channel.key$validFrom~validUntil'
```

Comment:
The only thing that might be controversial is this little change. `oneLineTrim` would remove the empty space between these two as it's written on a new line. 

However, intuitively I think `@` is the right choice here.